hoodie-client/src/test/java/com/uber/hoodie/index/bloom/TestHoodieBloomIndex.java

```diff
@@ -37,29 +36,21 @@
 import com.uber.hoodie.common.table.HoodieTableMetaClient;
 import com.uber.hoodie.common.util.FSUtils;
 import com.uber.hoodie.common.util.HoodieAvroUtils;
-import com.uber.hoodie.config.HoodieStorageConfig;
 import com.uber.hoodie.config.HoodieWriteConfig;
-import com.uber.hoodie.io.storage.HoodieParquetConfig;
-import com.uber.hoodie.io.storage.HoodieParquetWriter;
 import com.uber.hoodie.table.HoodieTable;
+
```

Comment:
May be try to avoid reshuffle of these imports ?

Reply:
sure.